CONTRIBUTING.md

```diff
@@ -29,6 +29,12 @@ yarn install
 
 This will download and install all packages needed.
 
+```sh
```

Comment:
A cosmetic change: I don't always remember to build, so I thought it might be noted in the instructions.